great_ape_safe/ape_api/cow.py

```diff
@@ -68,14 +62,30 @@ def _sell(self, sell_token, mantissa_sell, buy_token,
         print('')
         assert r.ok and r.status_code == 200
 
+        return r.json()
+
+
+    def _sell(self, sell_token, mantissa_sell, buy_token,
+        mantissa_buy, deadline, coef, destination, origin):
+        """call api to get sell quote and post order"""
+
+        # set destination to self if not specified
+        destination = self.safe.address if not destination else destination
+
+        # make sure mantissa is an integer
+        mantissa_sell = int(mantissa_sell)
+
+        # get the fee and exact amount to buy after fee
+        fee_and_quote = self.get_fee_and_quote(sell_token, buy_token, mantissa_sell)
+
         # grab values needed to post the order to the api
-        fee_amount = int(r.json()['fee']['amount'])
+        fee_amount = int(fee_and_quote['fee']['amount'])
         if mantissa_buy:
             # overwrite quote in case order has a limit
             mantissa_buy == int(mantissa_buy)
```

Comment:
```suggestion
            mantissa_buy = int(mantissa_buy)
```
old bug left unfixed :)